.golangci.yml

```diff
@@ -25,24 +25,19 @@ linters-settings:
   staticcheck:
     checks: [ "all" ]
   godot:
-    scope: declarations
+    scope: all
     capital: true
+    period: true
   goimports:
     local-prefixes: "github.com/auth0/terraform-provider-auth0"
-  revive:
-    rules:
-      - name: package-comments
-        severity: warning
-        disabled: true
```

Comment:
If we add settings in this section, it means that ONLY the settings we add will get executed, this was unintended. So we're moving the package comments to the excluded section below.